app.py

```
@@ -308,21 +123,30 @@ async def test_evaluate_text(request: TextEvaluationRequest, pronunciation_score
         history.append({"role":"assistant", "content":[{"type":"text","text":completion.to_json()}]})
         _next = json.loads(completion.choices[0].message.content)
-        print(_next)
+
+        # print(_next)
         print(type(_next))
         if "main_response" in _next:
             if "conversation" in _next.get("main_response"):
                 print("next_npc_ :", _next["main_response"]["conversation"][-1]["npc"])
+                next_npc_conversation = _next["main_response"]["conversation"][-1]["npc"]
+                tts_file_name = await convert_text_to_speech(next_npc_conversation)
+                if pronunciation_score != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)